backend/test/images/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
What's this do? Would a simple `.gitkeep` suffice with a corresponding entry in `backend/.gitignore`? fine either way

Reply:
We can actually probably just do with the git ignore all images. I just pulled that in from what I had in the /images.